cql3/statements/create_keyspace_statement.cc

```diff
@@ -215,6 +215,11 @@ create_keyspace_statement::execute(query_processor& qp, service::query_state& st
     });
 }
 
+lw_shared_ptr<keyspace_metadata> create_keyspace_statement::get_keyspace_metadata(const locator::token_metadata& tm) {
+    _attrs->validate();
```

Comment:
I hope this call is harmless. I remember in the past I saw this validate() function changing the place where the strategy class is stored, and don't remember if we have code that relies on running before the validate call and will now not find what it hopes to find.

Reply:
I need this `validate()` call exactly because it is the one storing the strategy class.